manageiq-operator/Makefile

```diff
@@ -119,7 +119,8 @@ run: manifests generate fmt vet ## Run a controller from your host.
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
 	echo "$(date +"%Y%m%d%H%M%S")-$(git rev-parse HEAD)" > ./BUILD
-	docker build -t ${IMG} .
+	cat ./BUILD
+	docker build -t ${IMG} . --pull --format docker
```

Comment:
This might affect s390x where we need buildah format

Reply:
Why do we need buildah format on s390x?

Reply:
Also, a few lines above we're tacking on `cmd+=" --format docker"` for s390x builds of the other images.

Reply:
Ah ok then I mixed it up... We tried a number of passes before finding the one that works